api/candlepin-api-spec.yaml

```diff
@@ -4217,6 +4414,8 @@ components:
 
     GuestIdDTOArrayElement:
       description: Represents a guest ID running on a virt host consumer. Does not include the attributes field
+      required:
```

Comment:
Same here, `required:` needs to be at the same level as `properties:` is

Reply:
Corrected.